src/stories/organisms/examples/examples.knobs.stories.js

```diff
@@ -0,0 +1,8 @@
+import {text, object} from '@storybook/addon-knobs';
+import data from './examples.stories.json';
+
+export default () => ({
+    title: text('Examples - title', data.title),
+    horizontalList: object('Examples - horizontal List', data.horizontalList),
+    cards: object('Examples - cards', data.cards)
```

Comment:
In the 'examples' organism the knobs are fin, can you add knobs to the 'card' molecule as well? Containing an image, title, description and link.